src/canvas/quadFeature.js

```diff
@@ -0,0 +1,123 @@
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of class quadFeature
+ *
+ * @class
+ * @param {Object} arg Options object
+ * @extends geo.quadFeature
+ * @returns {geo.canvas.quadFeature}
+ */
+//////////////////////////////////////////////////////////////////////////////
+geo.canvas.quadFeature = function (arg) {
+  'use strict';
+
+  if (!(this instanceof geo.canvas.quadFeature)) {
+    return new geo.canvas.quadFeature(arg);
+  }
+  geo.quadFeature.call(this, arg);
+
+  var m_this = this,
+      s_exit = this._exit,
+      s_init = this._init,
+      s_update = this._update,
+      m_quads;
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Build this feature
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this._build = function () {
+    if (!m_this.position()) {
+      return;
+    }
+    m_quads = this._generateQuads();
+  };
```

Comment:
In the tileLayer code, we set a z value based on the level.  We can use this z value to sort the tiles here so that they get rendered low to high:
```
    if (m_quads.imgQuads) {
        m_quads.imgQuads.sort(function (a, b) {
          return a.pos[2] - b.pos[2];
        });
      }
```